examples/cookbook/design/themes/lib/main.dart

```diff
@@ -70,3 +113,34 @@ class MyHomePage extends StatelessWidget {
     );
   }
 }
+
+void theme(BuildContext context) {
+  // #docregion Theme
+  Theme(
+    // Create a unique theme with `ThemeData`.
+    data: ThemeData(
+        colorScheme: ColorScheme.fromSeed(
+      seedColor: Colors.pink,
+    )),
```

Comment:
To make it easier to see which properties are nested where, add a trailing comma:

```suggestion
    data: ThemeData(
      colorScheme: ColorScheme.fromSeed(
        seedColor: Colors.pink,
      ),
    ),
```